lib/Runtime/Base/WindowsGlobalizationAdapter.cpp

```diff
@@ -459,12 +459,6 @@ if (this->object) \
         }
     }
 
-    void WindowsGlobalizationAdapter::ResetTimeZoneFactoryObjects()
-    {
-        DetachAndReleaseFactoryObjects(timeZoneCalendar);
-        DetachAndReleaseFactoryObjects(defaultTimeZoneCalendar);
-    }
-
```

Comment:
These had no consumers in ChakraCore and only one use in ChakraFull, which didn't seem to be important.